documentation/configuration.md

```diff
@@ -21,5 +21,19 @@ metrics: {
             enabled: true
         }
     }
+
+    jolokia {
+      # Enable or disable the Jolokia-based metrics/v2 endpoint.
+      # Default is true.
+      enabled: false
+
+      # Configure any of the settings listed at:
+      #   https://jolokia.org/reference/html/agents.html#war-agent-installation
+      config: {
+        # Specify a custom security policy:
+        #  https://jolokia.org/reference/html/security.html
+        policyLocation: "file:///etc/puppetlabs/<service name>/jolokia-access.xml"
```

Comment:
The only thing I would say on this is that in the documentation linked, all of these parameters are listed under a table named `Servlet Init Parameters`, so naming this section something like `servlet-init-params` might actually signal that you can use anything in that list.